### PR TITLE
initial commit log in

### PR DESCRIPTION
**Pull request by theVivek-kumar**
login page done please review

### CODE REVIEW

File: login/login.html [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+                Login
+
+            </div>
+            <div class="navbar-icons">
+                <i class="fas fa-cart-arrow-down"></i>
+                <i class="fas fa-heart"></i>
+            </div>
+
+        </div>
+    </div>
```
Comment (pankaj-prog) [inline login/login.html]: use semantic tags, say header or nav here

File: login/login.css [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+
+.navbar-input-search {
+    align-items: center;
+    display: flex;
+
+
+
+
+
+}
```
Comment (pankaj-prog) [inline login/login.css]: remove empty spaces

File: login/login.html [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    <div class="container-login">
+        <div class="login-container-FP">
+            <div class="login-Fp-header">
+                <h2>Login</h2>
+            </div>
```
Comment (pankaj-prog) [inline login/login.html]: FP ? Use descriptive names.

File: login/login.html [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+            <label class="lable" for="input">Email adress</label>
+            <input class="input-lg" type="email" placeholder="Enter Email">
+            <label class="lable" for="input">Password</label>
+            <input class="input-lg" type="password" placeholder="************">
+            <div class="input-check-btn">
+                <input class="input-checkbox" type="checkbox">
+                <h3>Remember me</h3>
+                <span class="frg-paswrd"> Forgot your Password?</span>
+            </div>
+            <a class="Login-input" href="#">Login</a>
```
Comment (pankaj-prog) [inline login/login.html]: this should be login button  i guess, input denotes: area where user needs input something

**Review by pankaj-prog — CHANGES_REQUESTED** [review]
Provide a deploy link and also add the screenshot of what you have changed.

File: login/login.html (hunk 1 above)
Comment (theVivek-kumar) [reply login/login.html]: sure

File: login/login.css [hunk 5]
```diff
@@ -0,0 +1,173 @@
...
+
+.navbar-head-logo {
+    color: #ece9f2;
+    display: flex;
+    align-items: center;
+}
+
+.navbar-login-btn {
+    background-color: rgb(255, 196, 1);
```
Comment (Adarsh-Patel-Dev) [inline login/login.css:48]: use color variables instead of rgb /hex code

**Review by Adarsh-Patel-Dev — CHANGES_REQUESTED** [review]

File: login/login.css [hunk 6]
```diff
@@ -0,0 +1,173 @@
...
+    padding: .5rem;
+    color: #312546;
+    font-weight: 600;
+    font-size: 1.1rem;
+    margin: .3rem;
+    width: 8%;
```
Comment (Adarsh-Patel-Dev) [inline login/login.css]: use rem instead of %

**Review by Adarsh-Patel-Dev — CHANGES_REQUESTED** [review]

File: login/login.css [hunk 7]
```diff
@@ -0,0 +1,173 @@
...
+    justify-content: center;
+    margin-top: .6rem;
+}
+
+.navbar-input-search {
+    align-items: center;
+    display: flex;
+
+
+
```
Comment (Adarsh-Patel-Dev) [inline login/login.css]: remove extra spaces here

**Review by Adarsh-Patel-Dev — CHANGES_REQUESTED** [review]

**Pull request merged**